core/statix.multilang/src/test/resources/mb/statix/multilang/base/base.spec.aterm

```diff
@@ -0,0 +1,2 @@
+FileSpec(["imported"],[],[Decl()],EOP(),[Rule(NoName(),C("base!ok",[Wld(){OfSort(SCOPE())}]),CTrue())],[])
```

Comment:
How stable are these files? If Statix changes (and it will), would these need to be updated?

Reply:
That is a valid concern, I would need to ask Hendrik for that. I adopted this format because integrating the complete statix builder (partially written in Stratego) is quite heavy 

However, when the format changes, the `SpecUtils#loadSpec` (and especially `SpecUtils#fileSpec`) implementation would need to be adapted as well. I'll try to figure out if we can lift these responsibilities to the statix runtime.

Reply:
Reply from Hendrik:
> Het format van de ATerm files is tot nu toe vrij stabiel. (...) Dus gewoon met de hand die merged ATerm genereren is waarschijnlijk het makkelijkst om te kunnen focussen op de rest.
So I would say this is accetable.

Reply:
I see. But can't you access the spec files from the SDF3 example project, and test there instead? Those are generated from Statix. 

I would like to avoid hand generated files because they will break in the future and only you know how to generate them. In that case, we'd have to delete your tests.

Reply:
Yeah, the code this is testing is touched by the SDF3-tests as well, but more on an integration-level. Maybe its best to remove these tests, because I cannot think of a way to generate them currently.

Reply:
Let's just keep them for now. Can always remove them later if they are problematic.

Reply:
Ok :-)